Tom

tomwpaterson-shape opened the pull request.

alihejazi commented on src/globals/mixins.scss:
```diff
@@ -55,3 +66,18 @@
 }
+
+
+@mixin corner-clip-bottom-right() {
```
@tomwpaterson-shape this isn't used anywhere. You sure we need it? If not, would mind deleting it from the code?

tomwpaterson-shape commented:
HI
I have removed this. I added it in for completeness, but as you say, it's not used at the moment. Do you need me to do another pull request?
    On Thursday, 8 September 2022 at 17:01:27 BST, alihejazi ***@***.***> wrote:  
 
 


@alihejazi commented on this pull request.

In src/globals/mixins.scss:
> @@ -55,3 +66,18 @@
       0 90%      /* bottom left */
     );
 }
+
+
***@***.*** corner-clip-bottom-right() {

@tomwpaterson-shape this isn't used anywhere. You sure we need it? If not, would mind deleting it from the code?

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you were mentioned.Message ID: ***@***.***>

alihejazi commented:
> HI I have removed this. I added it in for completeness, but as you say, it's not used at the moment. Do you need me to do another pull request? On Thursday, 8 September 2022 at 17:01:27 BST, alihejazi ***@***.***> wrote: @alihejazi commented on this pull request. In src/globals/mixins.scss:
> @@ -55,3 +66,18 @@
> 0 90% /* bottom left */ ); } + + ***@***.*** corner-clip-bottom-right() { @tomwpaterson-shape this isn't used anywhere. You sure we need it? If not, would mind deleting it from the code? — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you were mentioned.Message ID: ***@***.***>

@tomwpaterson-shape Thanks. You don't need to make a new PR because this PR is automatically updated as soon as you commit and push your code changes to GitHub. I think I will leave this PR open until @NoelLH can take a look too, but it looks good so far

NoelLH commented on src/components/biggive-hero-image/biggive-hero-image.tsx:
```diff
@@ -0,0 +1,50 @@
+/* eslint-disable prettier/prettier */
```
@tomwpaterson-shape Why are turning off style rules as opposed to using the auto lint fix commands?

NoelLH commented on src/components/biggive-hero-image/test/biggive-hero-image.spec.tsx:
```diff
@@ -0,0 +1,36 @@
+/* eslint-disable prettier/prettier */
```
As above

NoelLH commented on src/components/biggive-hero-image/test/biggive-hero-image.spec.tsx:
```diff
@@ -0,0 +1,36 @@
+import { BiggiveHeroImage } from '../biggive-hero-image';
+
+/**
+ * @todo Ideally we should test *all* `@Prop`s and basics of SVG FontAwesome icon rendering.
```
Copy/paste not applicable here?

NoelLH commented on src/globals/mixins.scss:
```diff
@@ -7,6 +7,17 @@
 
 // Layout + spacing
+@mixin flex-container-row() {
+  display: -webkit-box;
```
I don't think these vendor specific selectors are useful these days? Let's avoid if it's not expanding browser support meaningfully so we don't make the components heavier than they need to be.

NoelLH commented on src/index.html:
```diff
@@ -12,7 +12,13 @@
 
+
+
+  <biggive-hero-image slug="My Slug" main-title="This is my title" teaser="this is the intro text which I want to add"></biggive-hero-image>
```
Suggest we start using Storybook. atm you and @alihejazi are both updating this sample file which is just going to mean merge conflict resolution work for no benefit, very shortly.

NoelLH commented on package-lock.json:
```diff
@@ -13582,15 +13582,15 @@
-      "version": "8.0.1",
-      "resolved": "https://registry.npmjs.org/husky/-/husky-8.0.1.tgz",
-      "integrity": "sha512-xs7/chUH/CKdOCs7Zy0Aev9e/dKOMZf3K1Az1nar3tzlv0jfqnYtu235bstsWTmXOR0EfINrPa97yy4Lz6RiKw==",
+      "version": "7.0.4",
```
I'm a bit concerned about this changing with component work as it could make dependency stuff very confusing and diffs messy.

Do you think it would be possible to update to a Node that is current/ receiving security updates so that we can all standardise on a newer `husky` version? https://nodejs.org/en/blog/release/v12.22.12/

I'm also not sure if this part of the lockfile would change with an `npm install`. Do you know if you ran this, or `update` @tomwpaterson-shape?

alihejazi commented:
@tomwpaterson-shape Thanks for this work, looks fantastic! If possible, can you please address the following issues:
1. `npm run build` fails. Can you please run it locally and fix the issues that are logged in the terminal, and then commit the fixes? This should also ensure that all the component readme files are up to date.
2. Lint checks are failing - are you able to fix the lining issues using Prettier with ESlint?
3. `npm run storybook` fails -> there's a good chance this might start passing if point 1. is resolved.

Thanks a lot for your work so far; I did a quick local run of `npm start` and it is looking awesome!

@NoelLH do you have any other comments?
